ez-navigation-hub.html

```diff
@@ -1,15 +1,33 @@
 <link rel="import" href="../polymer/polymer-element.html">
 
-<!--
-`ez-navigation-hub`
-
-
-@demo demo/index.html
--->
-
 <style>
     ez-navigation-hub {
         display: block;
     }
+
+    ez-navigation-hub .ez-zone-list,
```

Comment:
actually, there should pretty much no CSS here, except the link visibility handling (`ez-zone-link` and `is-in-active-zone`). The rest should be moved in hybrid-platform-ui (maybe later, since there's an unmerged PR about styling).